content/help/glossary/service-registry.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Service Registry
+---
+
+Istio maintains an internal service registry containing the set of [services](#service),
+and their corresponding [service endpoints](#service-endpoint), running in a service mesh.
+Most services are automatically added to the registry by platform-specific Pilot plug-ins,
```

Comment:
Pilot adapters (plug-ins have a different meaning for Pilot). Might also want to specify that Istio is merely reading the service registry to generate Envoy configuration. Its not providing service discovery (service discovery is the responsibility of the underlying platform (k8s/consul/plain DNS).